src/riscv_core/veer_el2/config/compile.yml

```diff
@@ -94,7 +94,7 @@ targets:
       - $COMPILE_ROOT/tb/ahb_sif.sv
     tops: [el2_veer_wrapper_tb]
   sim:
-    pre_exec: $TOOLS/scripts/run_test_makefile
+    pre_exec: $MSFT_TOOLS/scripts/run_test_makefile
```

Comment:
Just wondering if this is an intentional change?

Reply:
Yeah, this is to support regression pipelines that we're running on internal toolchains; the script referenced here exists in a different repository from this one.